examples/tail/main.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
Let's make this example program a bit more interesting, and use it with Freq(), as you suggested in the issue discussion. Use it to find the least frequent lines in... something.

Reply:
Will change the example as needed

Reply:
Should I create a new example which shows the working of Last along with Freq , because I think that he current example shows the simplest and the most common working of Last , as it emulates the working of tail command.

Reply:
What I've tended to do so far is put the minimal example in the README, but in the accompanying example programs, do some slightly more interesting things. After all, the chief value of `script` is the ability to use _combinations_ of methods; if we can produce some working examples which are as close to real sysadmin programs as possible, that's a good thing to aim for.